doc/preview_install_guide.md

```diff
@@ -185,15 +182,23 @@ curl -L https://aka.ms/ProjectAzInstall | sudo bash
 
 ## CoreOS Stable-899.15.0 / Beta-1010.1.0 / Alpha-1010.1.0
 
-Doesn't have python installed by default and is not currently supported.
+Doesn't have python installed by default and is not currently supported.  We recommend using [Docker](#docker) to access the CLI on this platform.
+
+## Docker
 
+> Note that our docker images contain the latest code from the master branch and **does not match** the 
+> same "partner build" as the other methods in this document.
+
+Run `docker run -it azuresdk/azure-cli-python:latest bash`
 
 ## Windows (cmd)
 
+The CLI is available for Windows though PIP using the steps below.  If you do not have Python/PIP installed, consider using [Docker](#docker) to access the CLI.
+
 Set the environment variable to point to the version you wish to install.
 
 ```shell
-    set AZURE_CLI_NIGHTLY_VERSION=2016.06.30.nightly
+    set AZURE_CLI_NIGHTLY_VERSION=2016.07.07.nightly
```

Comment:
The current install script points to 2016.06.30.
Should this be 2016.07.27 in preparation for tomorrow instead of 2016.07.07?